### DISCUSSION TITLE
is there anything like text() from addon-knobs available in addon-controls?

### DISCUSSION THREAD

**Discussion by andyford** [storybookjs/storybook#16091] [Help] [answered]
I'd like to allow Storybook users to edit arbitrary text in stories which are not defined as props at the top level of the current component. With addon-knobs, this (among other things) was available via the `text()` helper. Is there any equivalent to the knobs text() helper available via addon-controls or available in general?

**Comment by shilman** [comment, marked as answer]
```js
export default {
  title: 'Gizmo',
  component: Gizmo,
  argTypes: {
    blah: { type: 'string' },
  },
};
```

https://storybook.js.org/docs/react/essentials/controls

**Reply by andyford** [reply]
Thanks!  This works. I didn't glean this from the docs. And now that I know how to do it, it still feels like the docs don't cover this use case very clearly. If I find some time, I'll submit a PR to improve the docs
